CONTRIBUTING.md

```diff
@@ -81,36 +81,109 @@ If you want to contribute, but you are not sure where to start - look for [issue
+```
+or
 ```
-tag-version-prefix=""
-message="release: cut the %s release"
+npm --no-git-tag-version version [version] --allow-same-version -m "release: cut the %s release"
 ```
+NOTE: Check the changelog!!!
 
-3. Create new branch for the release:
-```bash
-git checkout -b username/release-version
+7. Create release-branch with change log
+```
+git checkout -b release-[version]
 ```
 
-4. Execute [`npm version`](https://docs.npmjs.com/cli/version) to bump the version in the `package.json` file, tag the release and update the CHANGELOG.md:
-```bash
-npm version [patch|minor|major]
+7. Add changes
+```
+git add changed-files
+git commit -m "release: cut the %s release"
+git push
+```
+8. Create git tag version with format 0.22.3
 ```
+git tag version
+git push --tags
+```
+9. Create a pull request from git in web or try to use script below. Be careful to base your branch on the correct "release" branch
+```
+curl -d '{"title": "release: cut the [version] release","body": "docs: update changelog","head": "[BRANCH]","base": "release"}' -X POST https://api.github.com/repos/NativeScript/nativescript-dev-webpack/pulls -H "Authorization: token ${GIT_TOKEN}"
+```
+10. Merge PR into release branch.
 
-5. Push all the changes to your branch and create a pull request:
-```bash
-git push --set-upstream origin username/release-version --tags
+11. If all checks has passed publish package. Usually the night builds will be triggered and the package will be ready to be released on the next day. 
+12. Don't forget to tag the release branch 
+```
+git tag [version]
+git push tags
+```
+Tips to remove tags: 
+```
+git push --delete origin [version]
+git tag -d [version]
 ```
 
-6. Publish the package to `npm` after the pull request is merged:
-```bash
-npm publish
+## Merge changes from release into master
+
+![](./merge-guidance-schema.png)
+
+### Here are steps described in the diagram above.
+
+1. Make sure you are in release branch:
+```
+git checkout release
+git pull
+```
+2. Create PR to merge changes back in master and preserve history:
+```
+git checkout -b merge-release-in-master-branch
+git push --set-upstream origin merge-release-in-master-branch
+git merge origin/master
+```
+3. Resolve conflicts. Choose to keep the version of master branch. If it is needed to revert versions of modules, see at the bottom.
+
+4. Add conflicts:
+```
+git add resolved files
+```
+5. Commit changes with default merge message:
+```
+git commit
+git push
+```
+
+6. Create pull request. Replace replace env merge-release-in-master-branch with its value
+```
+curl -d '{"title": "chore: merge release in master","body": "chore: merge release in master","head": "merge-release-in-master","base": "master"}' -X POST https://api.github.com/repos/NativeScript/NativeScript/pulls -H "Authorization: token ${GIT_TOKEN}"
+```
+
+**If needed, revert version of modules and platform declarations to take the one from master:**
```

Comment:
don't forget to bump release and master versions
for example after releasing 7.3.4 from release, update master version to 7.4.0, merge master in release, update release version to 7.3.5